### PR TITLE
fixed full refactory

### PR DESCRIPTION
**Pull request by IQB696**

### CODE REVIEW

File: src/msgr/forms.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from .models import Task
+from django.forms import ModelForm, TextInput, Textarea
+
+
+class TaskForm(ModelForm):
```
Comment (JeffrysIT) [inline src/msgr/forms.py:5]: Why class is called "Task Form" not "Message Form"

File: src/msgr/forms.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    class Meta:
+        model = Task
+        fields = ["title", "task"]
```
Comment (JeffrysIT) [inline src/msgr/forms.py:8]: its not task. Its "message"

File: src/msgr/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Task',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=50, verbose_name='Name: ')),
+                ('task', models.TextField(verbose_name='Option: ')),
```
Comment (JeffrysIT) [inline src/msgr/migrations/0001_initial.py:19]: we are have any validation params on null for these fields?

File: src/msgr/migrations/0002_alter_task_options.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('msgr', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='task',
+            options={'verbose_name': 'Задача', 'verbose_name_plural': 'Задачи'},
```
Comment (JeffrysIT) [inline src/msgr/migrations/0002_alter_task_options.py:15]: Why we are have any russian symbols?

File: src/msgr/models.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Task(models.Model):
+    title = models.CharField('Name: ', max_length=50)
+    task = models.TextField('Option: ')
```
Comment (JeffrysIT) [inline src/msgr/models.py:6]: same. Why it's a task?

File: src/msgr/models.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+    def __str__(self):
+        return self.title
+
+    class Meta:
+        verbose_name = 'Задача'
+        verbose_name_plural = 'Задачи'
```
Comment (JeffrysIT) [inline src/msgr/models.py:13]: russian again

File: src/msgr/templates/msgr/base.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+<body>
+    <div class="d-flex flex-column flex-md-row align-items-center pb-3 mb-4 border-bottom">
+      <a href="/" class="d-flex align-items-center text-dark text-decoration-none">
+        <span class="fs-4">ProjectSite</span>
+      </a>
+
+      <nav class="d-inline-flex mt-2 mt-md-0 ms-md-auto">
+        <a class="me-3 py-2 text-dark text-decoration-none" href="{% url 'home' %}">Главная</a>
+        <a class="me-3 py-2 text-dark text-decoration-none" href="{% url 'about' %}">Про нас</a>
+        <a class="me-3 py-2 text-dark text-decoration-none" href="{% url 'create' %}">Добавить сообщение</a>
```
Comment (JeffrysIT) [inline src/msgr/templates/msgr/base.html:20]: could you rename all variables on english?

File: src/msgr/templates/msgr/create.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
+{% extends 'msgr/base.html' %}
+
+{% block title %}
+Введите сообщение
```
Comment (JeffrysIT) [inline src/msgr/templates/msgr/create.html:4]: Lets switch to the english. Full site

File: src/msgr/views.py [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+def about(request):
+    return render(request, 'msgr/about.html')
+
+
+def create(request):
+    error = ''
+    if request.method == 'POST':
+        form = TaskForm(request.POST)
+        if form.is_valid():
```
Comment (JeffrysIT) [inline src/msgr/views.py:19]: do a real validation. Just not null check and can't hold the numbers in the author field for example

File: src/msgr/migrations/0001_initial.py (hunk 3 above)
Comment (IQB696) [reply src/msgr/migrations/0001_initial.py:19]: It's the migration to databases, better to don't change anything to don't broke.

**Pull request merged**